gammapy/cube/core.py

```diff
@@ -324,6 +320,22 @@ def solid_angle_image(self):
         sky_map = SkyMap.from_image_hdu(image_hdu)
         return sky_map.solid_angle()
 
+    def sky_image(self, slicepos):
+        """Slice 3-dim cube into a 2-dim `~gammapy.image.SkyMap along slicepos`.
```

Comment:
@cdeil Would it be okay to call SkyMap.copy() on a newly created copy instead of working with each parameter and determining if it's mutable? 
I think the overhead is negligible as SkyMap.__init__ consist of just several "=" operators